miden-lib/asm/miden/account.masm

```diff
@@ -86,22 +86,19 @@ end
 
 #! Sets an item in the account storage. Panics if the index is out of bounds.
 #!
-#! Stack: [index, V']
+#! Stack: [index, V', 0, 0, 0]
```

Comment:
this is because of https://github.com/0xPolygonMiden/miden-base/issues/783

Reply:
As I mentioned in the comment for #783, I'm not sure I understood the motivation for this change. I think it was working as intended (e.g., the user would be able to do `exec.account::set_item` and it would not insert any extra zeros into the stack. Or am I missing something?